qucs/qucs/qucs.cpp

```diff
@@ -1793,7 +1793,7 @@ void QucsApp::slotRefreshSchPath()
   this->updateSchNameHash();
   this->updateSpiceNameHash();
 
-  int exit = QMessageBox::information(this, tr("Update schematic path"),
+  QMessageBox::information(this, tr("Update schematic path"),
       tr("The schematic file path has been refreshed."), tr("OK"));
```

Comment:
I do not see a reason for this message box. Of all the other software I have used, "Save As" is supposed to change to a different filename. Or am I missing a point?